Trading/Mode/remote_trading_signals_trading_mode/remote_trading_signals_trading.py

```diff
@@ -0,0 +1,469 @@
+        return personal_data.TraderOrderTypeClasses[order_type] == order.__class__
+
+    def _parse_signal_orders(self, signal):
+        to_create_orders = []
+        to_edit_orders = []
+        to_cancel_orders = []
+        to_group_orders = []
+        for order_description in signal.orders:
+            action = order_description.get(trading_enums.TradingSignalOrdersAttrs.ACTION.value)
+            if action == trading_enums.TradingSignalOrdersActions.CREATE.value:
+                to_create_orders.append(order_description)
+            elif action == trading_enums.TradingSignalOrdersActions.EDIT.value:
+                to_edit_orders.append(order_description)
+            elif action == trading_enums.TradingSignalOrdersActions.CANCEL.value:
+                to_cancel_orders.append(order_description)
+            elif action == trading_enums.TradingSignalOrdersActions.ADD_TO_GROUP.value:
+                to_group_orders.append(order_description)
+        return to_create_orders, to_edit_orders, to_cancel_orders, to_group_orders
+
+
+class RemoteTradingSignalsModeProducer(trading_modes.AbstractTradingModeProducer):
+
+    def get_channels_registration(self):
+        # trading mode is waking up this producer directly from signal channel
+        return []
+
+    async def signal_callback(self, signal):
+        exchange_type = signal.exchange_type
+        if exchange_type == trading_signals.get_signal_exchange_type(self.exchange_manager).value:
+            state = trading_enums.EvaluatorStates(signal.state)
+            await self._set_state(self.trading_mode.cryptocurrency, self.trading_mode.symbol, state, signal)
+        else:
+            self.logger.error(f"Incompatible signal exchange type: {exchange_type} "
+                              f"with current exchange: {self.exchange_manager}")
+
+    async def _set_state(self, cryptocurrency: str, symbol: str, new_state, signal):
+        self.state = new_state
+        self.logger.info(f"[{symbol}] update state: {self.state.name}")
+        # call orders creation from consumers
+        await self.submit_trading_evaluation(cryptocurrency=cryptocurrency,
+                                             symbol=symbol,
+                                             time_frame=None,
+                                             final_note=self.final_eval,
+                                             state=self.state,
+                                             data=signal)
+
+        # send_notification
+        if not self.exchange_manager.is_backtesting:
+            await self._send_alert_notification(symbol, signal)
+
+
+    async def _send_alert_notification(self, symbol, signal):
+        try:
+            import octobot_services.api as services_api
+            import octobot_services.enums as services_enum
+            title = f"New trading signal for {symbol}"
+            #todo
+            created = 0
+            edited = 0
+            cancelled = 0
```

Comment:
How can we set those values?

Reply:
Oh, thanks for seeing this !

Reply:
![image](https://user-images.githubusercontent.com/9078616/164910172-ca6dbfb9-2817-455e-942e-1a95854a44f5.png)
![image](https://user-images.githubusercontent.com/9078616/164910199-af574706-3b8a-4af0-80fe-9998e40e997b.png)